ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,658 @@
+                const titleY = chart.title.alignOptions.y,
+                    titleHeight = chart.title.alignOptions.height;
+
+                calcPosition.y += titleY + titleHeight;
+            }
+            if (chart.subtitle) {
+                const subtitletitleHeight = chart.subtitle.alignOptions.height;
+
+                calcPosition.y += subtitletitleHeight;
+            }
+            if (chart.rangeSelector && chart.rangeSelector.group) {
+                const rangeSelectorY = chart.rangeSelector.group.getBBox().y,
+                    rangeSelectorHight = chart.rangeSelector.group.getBBox().height;
+
+                calcPosition.y += (rangeSelectorY && rangeSelectorHight) ? rangeSelectorY + rangeSelectorHight : 0;
+            }
+        } else if (breadcrumbsOptions.position &&
+                    breadcrumbsOptions.position.verticalAlign === 'bottom' &&
+                    chart.marginBottom) {
+            if (chart.legend && chart.legend.group && chart.legend.options.verticalAlign === 'bottom') {
+                calcPosition.y -= chart.marginBottom -
+                        this.breadcrumbsGroup.getBBox().height -
+                        chart.legend.legendHeight;
+            } else {
+                calcPosition.y -= this.breadcrumbsGroup.getBBox().height;
+            }
+        }
+
+        if (breadcrumbsOptions.position && breadcrumbsOptions.position.align === 'right') {
+            calcPosition.x -= this.breadcrumbsGroup.getBBox().width;
+        } else if (breadcrumbsOptions.position && breadcrumbsOptions.position.align === 'center') {
+            calcPosition.x -= this.breadcrumbsGroup.getBBox().width / 2;
+        }
+        this.breadcrumbsGroup
+            .align(
+                calcPosition,
+                true,
+                chart.spacingBox
+            );
+    }
+
+    /**
+     * This method creates an array of arrays containing a level number
+     * with the corresponding series/point.
+     *
+     * @requires  modules/drilldown
+     *
+     * @function Highcharts.Chart#createList
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public createList(this: Breadcrumbs): void {
+        const chart = this.chart,
+            drilldownLevels = chart.drilldownLevels,
+            initialSeriesLevel = -1;
+
+        let breadcrumbsList: Array<any> = this.breadcrumbsList || [],
+            // If the list doesn't exist treat the initial series
+            // as the current level- first iteration.
+            currentLevelNumber: number = breadcrumbsList.length ?
+                breadcrumbsList[breadcrumbsList.length - 1][0] : initialSeriesLevel;
+
+        if (drilldownLevels && drilldownLevels.length) {
+            // Add the initial series as the first element.
+            breadcrumbsList = !breadcrumbsList.length ?
+                [[initialSeriesLevel, drilldownLevels[0].seriesOptions]] : breadcrumbsList;
```

Comment:
It looks that the `currentLevelNumber` from `339` is never used as here we use `initialSeriesLevel` and in `351` we are editing `currentLevelNumber`.

Reply:
In `350` we are using `currentLevelNumber` to compare levels, but in this case (going only one level down) I agree that the `351` line is not necessary.